src/applications/vaos/tests/appointment-list/redux/actions.unit.spec.js

```diff
@@ -37,7 +37,6 @@ import {
 } from '../../../utils/constants';
 import { STARTED_NEW_APPOINTMENT_FLOW } from '../../../redux/sitewide';
 
-import facilityData from '../../../services/mocks/var/facility_data.json';
```

Comment:
this change is unrelated to my work, but was causing a linting error (it's an unused variable)

Reply:
Should be fixed in master now

Reply:
Yep! I rebased and removed this commit. Thanks!